src/target/spi.c

```diff
@@ -67,9 +67,13 @@ void bmp_spi_read(const spi_bus_e bus, const uint8_t device, const uint16_t comm
 	bmp_spi_setup_xfer(bus, device, command, address);
 	/* Now read back the data that elicited */
 	uint8_t *const data = (uint8_t *const)buffer;
+#if 0
```

Comment:
Yes, I felt like `PLATFORM_HAS_SPI` and `PLATFORM_HAS_SPI_BLOCKWISE` or so would be nice to add. The first macro would guard dummy impls in all but two platforms, the second macro would dispatch to calling a block xfer function instead of slow byte wise callchains.
But first I wanted to evaluate flash size increase from this feature.